signer/src/stacks/contracts.rs

```diff
@@ -286,14 +287,14 @@ impl AsContractCall for CompleteDepositV1 {
     /// fortunate, because even if we wanted to, the only view into the
     /// stacks-core mempool is through the `POST /new_mempool_tx` webhooks,
     /// which we do not currently ingest.
-    async fn validate<S>(&self, db: &S, ctx: &ReqContext) -> Result<(), Error>
+    async fn validate<S>(&self, db: &S, req_ctx: &ReqContext) -> Result<(), Error>
     where
-        S: DbRead + Send + Sync,
+        S: Context + Send + Sync,
```

Comment:
`S` -> `C` (like you did above)

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/573/commits/5282a4f33cf0bafb97a5d49624e78db80632c4cb.